### DISCUSSION TITLE
forecaster.py

### DISCUSSION THREAD

**Discussion by gkumarg** [ourownstory/neural_prophet#198] [Q&A - get help using NeuralProphet] [answered]
Been getting an error since the latest update in 0.2.7
File "C:\Users\xx\anaconda3\envs\neural_prophet\lib\site-packages\neuralprophet\forecaster.py", line 7
    <!DOCTYPE html>
    ^
SyntaxError: invalid syntax

Anyone else getting this?j

**Comment by ourownstory** [comment]
Odd. line 7 in forecaster.py is:
`from torch.utils.data import DataLoader`

**Comment by ourownstory** [comment]
TBH, we have not done any testing with anaconda, but only in pip. 
If you can't get it to work in anaconda, try pip. 
If you figure out a solution, or would like to help us support anaconda, please share your discoveries!

**Comment by gkumarg** [comment]
I did install it through pip and it showed successful installation of neuralprophet 0.2.7.
Looking at the forecaster.py file in site-packages, it looks like it is not downloading that file correctly and just has a bunch of html stuff.
line 7 for me is <!DOCTYPE html>
Will keep checking to see if I can figure out a solution.

**Comment by gkumarg** [comment, marked as answer]
Somehow the latest files were not updating in my installation. I deleted my old fork and reforked from your repository again and also reinstalled neuralprophet. This time it brought in all the correct files. It's working now.

**Reply by ourownstory** [reply]
Wonderful, good to hear!
